ankerctl.py

```diff
@@ -224,6 +224,7 @@ def pppp_lan_search(env):
     """
     api = cli.pppp.pppp_open_broadcast(dumpfile=env.pppp_dump)
     try:
+        api.state = PPPPState.Connecting
```

Comment:
This change should go in `pppp_open_broadcast()`, otherwise fine :+1:

Reply:
Ok, I can move it there :)

Reply:
✅